src/components/Home/HomeSignUpSection.js

```diff
@@ -0,0 +1,49 @@
+import { useResponsive } from 'hooks/useResponsive'
+import { Box, Heading, Paragraph } from 'grommet'
+import { Button } from 'components/shared/Button'
+import { Column } from 'components/shared/Column'
+import { FixedContainer } from 'components/shared/FixedContainer'
+import { Row } from 'components/shared/Row'
+import { TextInput } from 'components/shared/TextInput'
+
+export const HomeSignUpSection = () => {
+  const { setResponsive } = useResponsive()
+  return (
+    <Box
+      background="gradient_blue_dark"
+      pad={{ vertical: setResponsive('medium', 'basex7', 'basex18') }}
+    >
+      <FixedContainer align="center">
+        <Heading level={1} color="white" margin={{ bottom: 'medium' }}>
+          Sign Up for Updates
+        </Heading>
+        <Paragraph
+          color="white"
+          margin={{ bottom: 'small' }}
+          textAlign="center"
+          size="large"
+        >
+          Be the first one to know about new features, compendia releases and
+          more!
+        </Paragraph>
+        {/* fixed width to preserve UI layout in wider screens */}
+        <Row width="500px">
```

Comment:
This value is for a fixed width (both min and max need to be 500px) for `large` devices. If we use max-only, then this UI element width can't maintain the minimum 500px (the sign up input field will shrink). Let me know your thought on this.

Also please note that from PR [35 - Implement Compendia Page](https://github.com/AlexsLemonade/refinebio-web/pull/39), the shared component [SignUpBlock](https://github.com/AlexsLemonade/refinebio-web/pull/39/commits/a1a0fd35707e1be65fb312dd9179a5cea20577f4) was added and this component is no longer used. Due to this, any adjustments for this will be made in the above-mentioned PR, thank you David!